app/models/registers.rb

```diff
@@ -0,0 +1,5 @@
+class Registers < ApplicationRecord
```

Comment:
Oh actually, I left it there because I didnt know how you were going to be doing auth since the control stuff was previously in here. Can we take it out.